src/main/java/chess/domain/piece/Color.java

```diff
@@ -0,0 +1,8 @@
+package chess.domain.piece;
+
+public enum Color {
+
+    BLACK,
+    WHITE,
+    ;
```

Comment:
다른 분들도 세미콜론을 내려 쓰시던데 이렇게 사용하라고 가이드가 있나요??🤔
```suggestion
    WHITE;
```

Reply:
이전 리뷰 중 다른 Enum값이 추가될 경우를 생각해서 저런 형태로 쓰곤 한다고 들었습니다.
물론 `Color` 가 추가될 상황은 존재하지 않겠지만, 변경 가능성이 높다고 생각되는 `GameCommand` 과의 통일성을 위해 위와 같이 작성했습니다!